DFC.ServiceTaxonomy.Editor/Program.cs

```diff
@@ -16,7 +23,21 @@ public static IHost BuildHost(string[] args)
             return Host.CreateDefaultBuilder(args)
                 .ConfigureLogging(logging => logging.ClearProviders())
                 .ConfigureWebHostDefaults(webBuilder => webBuilder
-                        .UseNLogWeb()
+                        .UseNLog()
+                        .ConfigureAppConfiguration((context, configuration) =>
+                        {
+                            LayoutRenderer.Register<TenantLayoutRenderer>(TenantLayoutRenderer.LayoutRendererName);
+
+                            var logConfigPath = Environment.GetEnvironmentVariable("ASPNETCORE_ENVIRONMENT") == Environments.Development
```

Comment:
Just a warning that the dev environment (at least) reports that it's the production environment.
Shouldn't affect this check, just something to keep in mind.